spec/cell_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'cell'
+
+describe Cell do
+
+  let(:cell) { Cell.new }
+  describe '#initialize' do
+    it 'defaults to correct attributes when passed nothing' do
+      expect(cell.value).to eq(0)
+      expect(cell.is_a_mine).to be false
+      expect(cell.is_visible?).to be false
+      expect(cell.adjacent_mines).to eq(0)
+      expect(cell.flagged).to be false
+    end
```

Comment:
This is actually great. I didn't know you could have multiple *expect* calls in the same it block.